.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,70 @@
+name: 🐞 Bug
+description: Create a report to help us improve
+title: "<title>"
+labels: ["Bug"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: textarea
+    attributes:
+      label: kamu version
+      description: Please copy and paste `kamu version` output.
+      placeholder: |
+        $ kamu version
+        appVersion: 0.150.0
+        buildTimestamp: 2023-12-29T10:41:15.288818966Z
+        ...
+      render: sh
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. In this environment...
+        2. With this config...
+        3. Run '...'
+        4. See error...
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Current Behavior
+      description: A concise description of what you're experiencing.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Logs
+      description: |
+        Please attach `.kamu/run/kamu.log` right after the execution of faulty command.
+        Another option: please copy and paste all relevant log data.
```

Comment:
`Alternatively, please copy and paste all relevant log data`

Reply:
Will be addressed in the follow-up PR

Reply:
Fixed in https://github.com/kamu-data/kamu-cli/pull/422